### PR TITLE
Add YouTube video support to articles.

### LINKED ISSUE #479: Add support for videos in news articles
We have support for images but supporting a video would be nice. We have sponsors who wants to send out a video to everyone. The video should be viewable directly from the feed without having to open the article.

### PR DESCRIPTION
**Pull request by RufusKoggRojder**
Fixes #479. 

Decided to go with embedding YouTube videos because I found that video hosting is very troublesome thanks to minio size limits.

Added [this dependency](https://www.npmjs.com/package/svelte-youtube-embed) for embedding.

### CODE REVIEW

File: src/routes/(app)/news/Article.svelte [hunk 1]
```diff
@@ -4,11 +4,14 @@
...
   import { languageTag } from "$paraglide/runtime";
   import { getFileUrl } from "$lib/files/client";
+  import Youtube from "svelte-youtube-embed";
 
   export let article: Article;
 </script>
 
-{#if article.imageUrl}
+{#if article.youtubeUrl}
+  <Youtube id={article.youtubeUrl.split("=")[1]} />
```
Comment (trilleplay) [inline src/routes/(app)/news/Article.svelte:13]: just use an iframe

File: src/routes/(app)/news/ArticleForm.svelte [hunk 2]
```diff
@@ -117,6 +128,14 @@
     accept="image/*"
   />
 
+  <FormInput
+    {superform}
+    field="youtubeUrl"
+    label="Youtube video URL"
+    onChange={onVideoSelected}
+    accept="video/*"
```
Comment (trilleplay) [inline src/routes/(app)/news/ArticleForm.svelte:136]: den ska inte accepta en `video/*` det är en sträng?

**Review by trilleplay — CHANGES_REQUESTED** [review]

**Review by Joel-Liden — COMMENTED** [review]
looks reasonable, with Trille changes.

File: src/routes/(app)/news/ArticleForm.svelte (hunk 2 above)
Comment (RufusKoggRojder) [reply src/routes/(app)/news/ArticleForm.svelte:136]: Tack, missade detta.

File: src/routes/(app)/news/Article.svelte (hunk 1 above)
Comment (RufusKoggRojder) [reply src/routes/(app)/news/Article.svelte]: Ok.

**Pull request merged**